CHANGELOG.md

```diff
@@ -4,6 +4,129 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.7.0-rc2 - 2023-01-18
```

Comment:
Is this date (`2023-01-18`) a placeholder or a copy/paste error?

Reply:
fortunately the release process requires `+lint-changelog` to pass, which will error if the date isn't today's date.